nix/shell.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs ? (import <nixpkgs>) { } }:
+let
+  # `volatile` is not packaged in nix.
+  volatile = pkgs.python38Packages.buildPythonPackage rec {
+    pname = "volatile";
+    version = "2.1.0";
+    src = pkgs.python38Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "1lri7a6pmlx9ghbrsgd702c3n862glwy0p8idh0lwdg313anmqwv";
+    };
+    doCheck = false;
+  };
+  python = pkgs.python3.withPackages
+    (python-packages: with python-packages; [ click kubernetes volatile ]);
```

Comment:
It feels like this stuff should be in `default.nix`...

Reply:
There is no `default.nix` right now. We're pending a decision on how to tackle the nix setup in general, especially regarding the container.